README.md

```diff
@@ -5,7 +5,7 @@
         <p>ERP made simple</p>
     </p>
 
-[![Build Status](https://travis-ci.com/frappe/erpnext.svg)](https://travis-ci.com/frappe/erpnext)
+[![Build Status](https://travis-ci.org/frappe/erpnext.svg?branch=master)](https://travis-ci.org/frappe/erpnext)
```

Comment:
I believe the branch should be `develop`. 

Why change the URL to `.org`? This repo's builds run on `.com` anyway

Reply:
i have changed the build branch to `develop`. And the URL should be `.org`. After these changes the build badge is functioning properly as it is showing `build cancelled`.

Reply:
Why does it have to be `.org`? Please add any references to support your reasoning.

Reply:
`travisCI`'s site has url https://travis-ci.org/. Also you can check the documentation for [travisCI](https://docs.travis-ci.com/user/status-images/). Cleary in the example images the URL is `.org`

Reply:
@gavindsouza what's the status of this PR? Do i need to make more changes? or Is it good enought to be merged?

Reply:
@gavindsouza may i know the status of this PR? Do i need to make more changes? or Is it good enought to be merged?

Reply:
@krush11 Thanks for the references. The example you're referring to, was written [4 years ago](https://github.com/travis-ci/docs-travis-ci-com/blame/master/user/status-images.md#L38). https://travis-ci.org is set to shut down by the end of this year. 

Changing the badge URL to https://api.travis-ci.com/frappe/erpnext.svg?branch=develop works best IMO, since there wouldn't be any redirects either.


References:
- https://mailchi.mp/3d439eeb1098/travis-ciorg-is-moving-to-travis-cicom
- https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps

Reply:
Thanks @gavindsouza. Commited the required changes.

Reply:
Fix the sider and commit messages so that all the checks pass. 
https://api.travis-ci.com/frappe/erpnext => https://travis-ci.com/frappe/erpnext

Reply:
i dont know how to squash commits